live_betting/utils.py

```diff
@@ -0,0 +1,21 @@
+import json
+from typing import Tuple
+
+from selenium import webdriver
+
+
+def load_fb_credentials(credentials_file_path: str) -> Tuple[str, str]:
+    credentials = json.load(open(credentials_file_path, "rb"))
+    return credentials.get("username"), credentials.get("password")
+
+
+def write_xpath(driver: webdriver, xpath: str, text: str):
```

Comment:
Wouldn't it make more sense to make a wrapper class for the webdriver and add these methods to it?

Reply:
It would be better to have a package for it. Wrapper is just some so-so solution as is utils file.